src/_variables.scss

```diff
@@ -3,103 +3,222 @@
-$btn-danger-border: darken($btn-danger-bg, 20%) !default;
+$btn-info-bg:             $brand-info !default;
+$btn-info-border:         darken($btn-info-bg, 20%) !default;
 
-$pagination-color: $btn-default-color !default;
-$pagination-bg: $btn-default-bg !default;
-$pagination-border: $btn-default-border !default;
+$btn-warning-color:       #000 !default;
+$btn-warning-bg:          $brand-warning !default;
+$btn-warning-border:      darken($btn-warning-bg, 20%) !default;
+
+$btn-danger-bg:           $brand-danger !default;
+$btn-danger-border:       darken($btn-danger-bg, 20%) !default;
 
-$pagination-hover-color: $btn-default-color !default;
-$pagination-hover-bg: darken($btn-default-bg, 8%) !default;
-$pagination-hover-border: darken($btn-default-border, 12%) !default;
 
-$pager-border-radius: 4px !default;
-
-// Footnotes
-$spacing: .375em !default;
-$ddRtnWidth: 3.375em !default;
-
-// Calendar
-$cal-text-head: #fff !default;
-$cal-text-body: #000 !default;
-$cal-dark: #333 !default;
-$cal-medium: #555 !default;
-$cal-active: #ccc !default;
-$cal-days-link: #111 !default;
-
-//Multimedia Player
-$mm-progress-fg-color: #176ca7 !default;
-$mm-progress-bg-color: #fff !default;
-$mm-cc-fg-color: #fff !default;
-$mm-cc-bg-color: #000 !default;
-$mm-ctrl-fg-color: #fff !default;
-$mm-ctrl-bg-color: #3e3e3e !default;
-$mm-ctrl-outline-color: #4aafff !default;
-$mm-ctrl-cc-color: #4aafff !default;
-$mm-volume-bg-color: #aaa !default;
-$mm-volume-thumb-color: #fff !default;
+//== Media queries breakpoints
+//
+//## Define the breakpoints at which your layout will change, adapting to different screen sizes.
 
+// WET Custom
 // Additional Viewports outside of Bootstrap definitions
-$screen-xxs-max: 479px !default;
-$screen-lg-max: 1599px !default;
-$screen-xl-min: 1600px !default;
+
+// Extra extra small screen
+$screen-xxs-max:          479px !default;
+// Large screen
+$screen-lg-max:           1599px !default;
+// Extra large screen
+$screen-xl-min:           1600px !default;
+
+
+//== Pagination
+//
+//##
+
+// Bootstrap Overrides
+$pagination-color:        $btn-default-color !default;
+$pagination-bg:           $btn-default-bg !default;
+$pagination-border:       $btn-default-border !default;
+
+$pagination-hover-color:  $btn-default-color !default;
+$pagination-hover-bg:     darken($btn-default-bg, 8%) !default;
+$pagination-hover-border: darken($btn-default-border, 12%) !default;
+
+// WET Custom
+$pager-border-radius:     4px !default;
+
+
+//
+// WET CUSTOM PLUGINS VARIABLES
+//
+
+//== Calendar
+//
+//##
+
+$cal-text-head:           #fff !default;
+$cal-text-body:           #000 !default;
+
+$cal-dark:                #333 !default;
+$cal-medium:              #555 !default;
+
+$cal-active:              #ccc !default;
+$cal-days-link:           #111 !default;
+
+
+//== Footnotes
+//
+//##
+
+$spacing:                 .375em !default;
+$ddRtnWidth:              3.375em !default;
+
+
+//== Multimedia player
+//
+//##
```

Comment:
The `//##`? Those are just placeholders for comments for formatting consistency similar to the Bootstrap variables file. We can take them out if you'd like.